.github/dependabot.yml

```diff
@@ -0,0 +1,44 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
better biweekly. Weekly was too frequent

Reply:
Dependabot doesn't seem to have `biweekly` ([ref](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#interval)). So used a cronjob instead.